packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -145,9 +146,14 @@ class DeepLinksController extends DisposableController
   DeepLinksController() {
     addAutoDisposeListener(
       selectedAndroidVariantIndex,
-      _handleSelectedAndroidVariantIndexChanged,
+      _handleSelectedVariantIndexChanged,
     );
-    // TODO(Hangyujin): Add listerner for selectedIosConfigurationIndex.
+    if (FeatureFlags.deepLinkIosCheck) {
+      addAutoDisposeListener(
+        selectedIosConfigurationIndex,
+        _handleSelectedVariantIndexChanged,
```

Comment:
does using the same listener mean we will be doing unnecessary work to rebuild the android links when the ios variant changed or vise versa?

Reply:
updated